arraycontext/container/traversal.py

```diff
@@ -520,6 +619,72 @@ def to_numpy(ary: Any, actx: ArrayContext) -> Any:
     """
     return rec_map_array_container(actx.to_numpy, ary)
 
+
+def flatten_to_numpy(ary: ArrayOrContainerT, actx: ArrayContext) -> np.ndarray:
```

Comment:
Why not have this call `flatten`?

Reply:
I went ahead and removed to `_numpy` versions altogether in 7f3cb6f, since doing `actx.to_numpy(flatten(ary, actx))` isn't particularly verbose to me.

@thomasgibson This will probably break the code you currently have. Sorry!

Reply:
I anticipated this and preemptively changed it https://github.com/inducer/grudge/pull/154/commits/96d0bf314eb12dc2b3e11ad8b13488e0b6c213e5 :stuck_out_tongue_winking_eye:

Reply:
Removing is fine by me.